core/templates/pages/signup-page/signup-page.component.ts

```diff
@@ -154,7 +158,7 @@ export class SignupPageComponent {
   submitPrerequisitesForm(
       agreedToTerms: boolean,
       username: string,
-      canReceiveEmailUpdates: string): void {
+      canReceiveEmailUpdates: string | null): void {
```

Comment:
comment for null missing

Reply:
On top write explanation comment why this type may be null, do i need to write here as well?